test/services/identity_verification_method.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({ path: 'test/.env.test' });
```

Comment:
Curious why this is necessary now?

Reply:
`dynamodb-local` requires values for `AWS_`* env vars, I probably could have overridden them inline as well but figured sourcing from env vars is closer to how it will function in a live environment.